airbyte-integrations/connectors/source-mixpanel/unit_tests/conftest.py

```diff
@@ -0,0 +1,190 @@
+import pendulum
+import pytest
+
+
+@pytest.fixture(name="config")
```

Comment:
nit:
```suggestion
@pytest.fixture
```
Your fixture and your function share the same names, `pytest` will automatically resolve its fixture according to function name, so in this context it's redundant to define your fixture name.

Reply:
changed in [commit](https://github.com/airbytehq/airbyte/pull/11633/commits/569391a5c437021aa9df128581557b16e60a3704)